deny.toml

```diff
@@ -7,7 +7,8 @@ allow = [
     "Unlicense",
     "BSD-3-Clause",
     "0BSD",
-    "ISC"
+    "ISC",
+    "CC0-1.0"
 ]
```

Comment:
Why was this updated?

Reply:
Seems like a dependency changed and this license is actually being used. A few weeks ago the checker had me remove it because it was unused, but now it wanted me to add back. Not sure what happened.

This license is a public domain license so we can allow it.